src/sync/polling/updaters/splitChangesUpdater.ts

```diff
@@ -45,15 +45,29 @@ interface ISplitMutations {
   segments: string[]
 }
 
+/**
+ * If there are defined filters and one feature flag doesn't match with them, its status is changed to 'ARCHIVE' to avoid storing it
+ * If there are set filter defined, names filter is ignored
+ *
+ * @param featureFlag feature flag to be evaluated
+ * @param filters splitFiltersValidation bySet | byName
+ */
+function matchFilters(featureFlag: ISplit, filters: ISplitFiltersValidation) {
+  const { bySet: setsFilter, byName: namesFilter } = filters.groupedFilters;
+  if (setsFilter.length > 0) return featureFlag.sets && featureFlag.sets.find((featureFlagSet: string) => setsFilter.indexOf(featureFlagSet) > -1);
```

Comment:
@emmaz90, the logic is correct here. However, I now realized that we cannot use the `find` array method because it is not available in some browsers with which the SDK is compatible (https://caniuse.com/array-find). 

We should refactor this sentence to use either `indexOf` or the `find` utility function in `lang/index.ts` if it is suitable for the work.

Also, refactor the usage of `find` in the `splitFilters.ts` file (`configuredFilter` function).